.github/workflows/test.yml

```diff
@@ -181,13 +106,38 @@ jobs:
           pip install -U tox setuptools==44.0.0
       - name: Run tests
         run: tox
-      - name: Log Test Failure
-        uses: ros-tooling/action-cloudwatch-metrics@0.0.2
-        with:
-          status: 'failure'
-        if: failure() && github.event_name == 'schedule'
-      - name: Log Test Success
-        uses: ros-tooling/action-cloudwatch-metrics@0.0.2
-        with:
-          status: 'success'
-        if: success() && github.event_name == 'schedule'
+
+  log_workflow_status_to_cloudwatch:
+    runs-on: ubuntu-latest
```

Comment:
I think we should pin the version of Ubuntu for consistency, even if we're doing a container build.

Reply:
Repos in ros-tooling have been using ubuntu-latest, so thought of keeping it consistent here.